bin/nanocld/src/cli.rs

```diff
@@ -37,6 +37,15 @@ pub struct Cli {
   /// Group id
   #[clap(long, default_value = "0")]
   pub(crate) gid: u32,
+  /// Docker daemon ssl certificate
```

Comment:
It's not the docker daemon ssl keys it's our just remove the `Docker daemon` part in the commentary!

Reply:
I removed the `Docker daemon` part

Reply:
Thanks!